R/clusterExpUnitsWithScamp.R

```diff
@@ -141,7 +141,7 @@
 
 
 .clusterExpUnitsWithScamp <- function(projectPath,
-                                      nameOccurenceNum,
+                                      nameOccurenceNumber,
```

Comment:
@loganknecht , I hate to be that guy, but you're still missing an r in Occurrence